projects/netgrif-components-core/src/lib/data-fields/case-ref-field/case-ref-default/case-ref-default.component.ts

```diff
@@ -23,11 +24,15 @@ export abstract class AbstractCaseRefDefaultComponent extends AbstractCaseRefBas
         this._sub = this.dataField.valueChanges().subscribe(() => {
             this.createFilter(this.dataField.value.length > 0 ? this.dataField.value : '');
         });
+        this._subComp = this.dataField.componentChange$().subscribe(() => {
+            this.createFilter(this.dataField.value.length > 0 ? this.dataField.value : '');
```

Comment:
Duplicate code for both subscriptions, maybe refactor into function so that changes are always made for both. Same in other files where subComp was added

Reply:
done